templates/aspnet-core/files/Helpers/PlatformRelationship.cs

```diff
@@ -0,0 +1,92 @@
+using System;
```

Comment:
Shouldn't this whole thing be in a separate config reader library?

Reply:
Eventually sure we should make one. For now isn't this sufficient, provided a name other than "helper" is used?